styles/blog6.css

```diff
@@ -0,0 +1,17 @@
+/* img { 
+    width: 400px; 
+    height: 400px; 
+    object-fit: contain;
+}  */
+
+/* .col {
+    column-gap: 20px;
+    background-color: tomato;
+} */
+
+/* button {
+    background-color:gray;
+    color: white;
+} */
+
+/* d-flex justify-content-left */
```

Comment:
Don't submit a file that contains all commented out code (-1)

-----
Great work! (87/100)